### PR TITLE
(PR2022-36/layout-of-login) and configuration files

### PR DESCRIPTION
**Pull request by Neoclassic-alt**
PR2022-36 Changed login layout

### CODE REVIEW

File: src/modules/Auth/components/SignInForm.tsx [hunk 1]
```diff
@@ -90,22 +91,25 @@ export const SignInForm = () => {
...
-              <Box alignSelf="flex-end">
+              <Box
+                display="flex"
+                justifyContent="space-between"
+                flexDirection="row"
+              >
+                <Button
+                  variant="text"
+                  type="button"
+                  onClick={() => history.push('/signup')}
```
Comment (VKhayretdinov) [inline src/modules/Auth/components/SignInForm.tsx:102]: It's not the best choice to specify the route here as just a string. Please move it to the constant above and think perhaps you already have this route somewhere in your codebase, and you can reuse it.

**Review by VKhayretdinov — CHANGES_REQUESTED** [review]

File: src/modules/Auth/components/SignInForm.tsx (hunk 1 above)
Comment (Neoclassic-alt) [reply src/modules/Auth/components/SignInForm.tsx:102]: Where place file with constants?

**Review by VKhayretdinov — APPROVED** [review]

File: src/constants/constants.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export const MAIN_PAGE = '/';
+export const AUTH_PAGE = '/auth';
+export const SIGNUP_PAGE = '/signup';
```
Comment (VKhayretdinov) [inline src/constants/constants.ts:3]: Sorry, I missed it. I guess rename file to `routes`

File: package.json [hunk 3]
```diff
@@ -11,18 +11,19 @@
...
     "react-dom": "17.0.2",
-    "react-hook-form": "7.25.3",
+    "react-hook-form": "^7.25.3",
     "react-hot-loader": "4.13.0",
     "react-query": "3.9.4",
-    "react-router": "5.2.1",
     "react-router-dom": "5.3.0",
-    "recoil": "0.6.1"
+    "recoil": "0.6.1",
+    "yup": "^0.32.11"
```
[the remark this answers is not in the document]
Comment (Neoclassic-alt) [reply package.json:26]: I installed package Yup, used for validation.

**Pull request merged**